decidim/decidim
# Use left outer join instead of include in `with_any_category` scope

#### :tophat: What? Why?
Use `left_outer_joins` instead of `includes` in `with_any_category` scope to fix the query when adding an order.

#### :pushpin: Related Issues
- Fixes #11467

#### Testing
1. Go to a participatory space with a proposals component.
2. Go to the proposals component.
3. Filter by category.
4. Order by "most commented" or "most followed" and see it works.

:hearts: Thank you!
